starter-code/index.html

```diff
@@ -1,42 +1,95 @@
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
 
-HD Music
-Listen to music as if you
-were listening live
+<body>
+  <div class="container">
+    <nav class="nav-bar">
+      <a href="#"><img class="logo" src="./images/spotify-logo.png" alt="" srcset=""></a>
+      <ul class="nav">
+        <li><a href="#">Premium</a></li>
+        <li><a href="#">Discover</a></li>
+        <li><a href="#">Help</a></li>
+        <li><a href="#">Download</a></li>
+      </ul>
+    </nav>
+    <div class="banner">
+      <div class="title-text">
+        <h1>Music for everyone.</h1>
+        <h2 class="subtitle">Spotify is now free on mobile, tablet and computer.
+          Listen to the right music, wherever you are.</h2>
+      </div>
+    </div>
+    <div class="features">
+      <h3 class="section-title">
+        What’s on Spotify?
+      </h3>
+      <ul class="feature-list">
+        <li class="feature"><img src="images/music-icon.png" alt="Music">
+          <h4 class="feature-title">Millions of Songs</h4>
+          <p class="feature-description">There are millions of
+            songs on Spotify</p>
+        </li>
+        <li class="feature"><img src="images/high-quality-icon.png" alt="High quality">
+          <h4 class="feature-title">HD Music</h4>
+          <p class="feature-description">Listen to music as if you
+            were listening live</p>
+        </li>
+        <li class="feature"><img src="images/devices-icon.png" alt="Devices">
+          <h4 class="feature-title">Stream Everywhere</h4>
+          <p class="feature-description">Stream music on your
+            smartphone, tablet or
+            computer</p>
+        </li>
+      </ul>
+    </div>
+    <div class="website-functions">
+      <h3 class="functions-title">
+        It’s as yeezy as Kanye West.
+      </h3>
+      <div class="selling-points">
+        <ul class="selling-point-item">
+          <li class="selling-point">
+            <h3 class="selling-point-title">
+              Search
+            </h3>
+            <p class="selling-point-description">
+              Know what you want to listen to?
+              Just search and hit play.
+            </p>
+          </li>
+          <li class="selling-point">
+            <h3 class="selling-point-title">Browse
+            </h3>
+            <p class="selling-point-description">Check out the latest charts,
+              brand new releases and great
+              playlists for right now.</p>
+          </li>
+          <li class="selling-point">
+            <h3 class="selling-point-title">Discover
+            </h3>
+            <p class="selling-point-description">Enjoy new music every Monday
+              with your own personal playlist.
+              Or sit back and enjoy Radio.</p>
+          </li>
+        </ul>
+        <img src="images/spotify-icon-white.png" alt="Spotify logo" class="icon-white selling-point-item">
```

Comment:
That logo might have been easier to handle as a background-image of div.website-functions.